.github/workflows/etcd-defrag.yaml

```diff
@@ -0,0 +1,48 @@
+name: Publish
+
+on:
+- push
+- workflow_dispatch
+
+jobs:
+
+  publish-tagged:
+    if: |
+      startsWith(github.ref, 'refs/tags/v') &&
+      !github.event.pull_request.head.repo.fork
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "1.19"
```

Comment:
Please reuse the golang version defined in [go-version.yaml](https://github.com/ahrtr/etcd-defrag/blob/main/.github/workflows/go-version.yaml), please refer to [test.yaml](https://github.com/ahrtr/etcd-defrag/blob/main/.github/workflows/test.yaml).